PlayCover/Views/MainView.swift

```diff
@@ -127,6 +153,43 @@ struct MainView: View {
                 }
                 .background(SplitViewAccessor(sideCollapsed: $collapsed))
             }
+            .sheet(isPresented: $addFolderPresented) {
+                VStack {
+                    HStack {
+                        TextField(text: $newFolder, label: {Text("folder.textfield.name")})
+                            .frame(height: 40)
+                        Picker(selection: $selectedSymbol, label: Text("Icon")) {
+                            ForEach(foldersObject.icons, id: \.self) { icon in
+                                Image(systemName: icon)
+                            }
+                        }.fixedSize()
+                    }
+                    HStack {
+                        Spacer()
+                        Button(NSLocalizedString("button.OK", comment: ""), action: {
+                            foldersObject.addFolder(folder: newFolder, icon: selectedSymbol)
+                            selectedSymbol = "folder"
+                            addFolderPresented.toggle()
+                            self.selectedView = foldersObject.folders.last?.id.hashValue
+                        })
+                        .disabled(newFolder.isEmpty)
+                        .keyboardShortcut(.defaultAction)
+                        Button(NSLocalizedString("button.Cancel", comment: ""), action: {
+                            newFolder = ""
+                            selectedSymbol = "folder"
+                            addFolderPresented.toggle()
+                        })
+                        .keyboardShortcut(.cancelAction)
+                    }
+                }
+                .padding()
+                .frame(width: 600, height: 100)
```

Comment:
Can you add a `.padding()` after the `.frame` so that the sheet looks more natural? Same with the editor sheet inside `AppFolderView`.